ee/clickhouse/util.py

```diff
@@ -18,7 +18,6 @@
 
 
 class ClickhouseTestMixin(QueryMatchingTest):
-    RUN_MATERIALIZED_COLUMN_TESTS = True
```

Comment:
This is used, why remove this?

Reply:
There's a little bit of context in the [commit message](https://github.com/PostHog/posthog/pull/7505/commits/8dd6e2ece6841497cfec6da5b185025db4a45ea1)

I can add it back.